fdbserver/BackupWorker.actor.cpp

```diff
@@ -878,25 +878,19 @@ ACTOR Future<Void> uploadData(BackupData* self) {
 		int lastVersionIndex = 0;
 		Version lastVersion = invalidVersion;
 
-		if (self->messages.empty()) {
```

Comment:
I remembered this is probably an optimization. If correctness passes, I'm fine remove it.